src/test/java/com/iota/iri/APIIntegrationTests.java

```diff
@@ -3,26 +3,51 @@
 import com.google.gson.Gson;
 import com.google.gson.GsonBuilder;
 import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.config.HttpClientConfig;
+import com.jayway.restassured.specification.RequestSpecification;
 import org.junit.Test;
 
 import java.util.HashMap;
 import java.util.Map;
+
 import static org.hamcrest.Matchers.containsString;
-import static com.jayway.restassured.RestAssured.*;
 
 public class APIIntegrationTests {
 
-    private static final Gson gson = new GsonBuilder().create();
+    // no result should ever take a minute
+    private static final int SOCKET_TIMEOUT = 60_1000;
```

Comment:
The 1 slipped off?
You meant 60_000 I assume

Reply:
LOL. Yup. :D